### DISCUSSION TITLE
著作物を引用する際の注意の「ないよう」はそのままでもいいのかもしれませんが「内容」のような気がしました。

### DISCUSSION THREAD

**Discussion by TORIFUKUKaiou** [increments/qiita-discussions#525] [要望・アイデア・不具合]
いつもQiitaを便利に使わせていただいております。ありがとうーーーーッ！！！ございます、です。

[著作物を引用する際の注意](https://help.qiita.com/ja/articles/about-copyright)ページの話です。

> 割合として6:4であれば良いというわけではなく、全体を照らしてみた時に、割合が4であっても引用部分が主たるないようになっていないことが必要です。

この文の「ないよう」は「内容」のほうがいいのかもしれないなあという気がしました。私がものを知らないだけで「ないよう」のままのほうがよいのかもしれません。
気がしただけですので私の勘違いでしたらCloseしてください。

**Comment by tanaka-asami415** [comment]
@TORIFUKUKaiou 
こちらこそいつもQiitaをご利用いただきありがとうございます!!!🙇

> この文の「ないよう」は「内容」のほうがいいのかもしれないなあという気がしました。

！！😱 教えてくださりありがとうございますッ！🙇
漢字の「内容」の方が適切なので修正します！

**Reply by tanaka-asami415** [reply]
@TORIFUKUKaiou 
こちら修正が完了しました！🙇 教えていただきありがとうございました！
https://help.qiita.com/ja/articles/about-copyright

**Comment by TORIFUKUKaiou** [comment]
ご連絡ありがとうーーーーッ！！！ございます！
